stories/bar/bar.stories.js

```diff
@@ -6,8 +6,39 @@ import '../../dist/segmented-button.css';
 export default {
     title: 'Components/Bar',
     parameters: {
-        description: `The Bar component is a container that holds titles, buttons and input controls. Its content is distributed in three areas - left, middle and right.
-The Bar has 2 modes - Desktop (default) and Tablet/Mobile (cozy).`,
+        description: `The bar component is a container that holds titles, buttons and input controls. The contents are distributed into three areas: left, middle and right. This component’s primary function is to display page **headers** and **footers**. It is mainly used to construct a **Page**, and acts as a building block for other components like **Dialog**, **Popover** etc.
```

Comment:
Its contents

Reply:
"Its" or "The" both work, so I can agree with this comment.